src/main/kotlin/no/nav/faktureringskomponenten/controller/TotalBeløpController.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.faktureringskomponenten.controller
+
+import no.nav.faktureringskomponenten.controller.dto.BeregnTotalBeløpDto
+import no.nav.faktureringskomponenten.service.beregning.BeløpBeregner
+import no.nav.security.token.support.core.api.Protected
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.BindingResult
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.*
+
+@Protected
+@Validated
+@RestController
+@RequestMapping("/totalbeloep")
```

Comment:
Liker konvensjonen som bruker navn for REST ressurs og verb for funksjoner.
F.eks. /totalbeloep/beregn. Ikke så viktig her

Reply:
Enig. Fikser det